SlowCheetah.VisualStudio/SlowCheetahPackage.cs

```diff
@@ -399,29 +421,33 @@ private void OnPreviewTransformCommand(object sender, EventArgs e)
         /// <param name="hierarchy">Current project hierarchy</param>
         /// <param name="parentId">Parent ID of the file.</param>
         /// <param name="transformName">Name of the transformation file</param>
+        /// <param name="docId">ID of the file to transform</param>
         /// <param name="documentPath">Resulting path of the file to transform</param>
         /// <returns>True if the correct file was found</returns>
-        private bool TryGetFileToTransform(IVsHierarchy hierarchy, uint parentId, string transformName, out string documentPath)
+        private bool TryGetFileToTransform(IVsHierarchy hierarchy, uint parentId, string transformName, out uint docId, out string documentPath)
         {
             IVsProject project = (IVsProject)hierarchy;
 
             IEnumerable<string> configs = ProjectUtilities.GetProjectConfigurations(hierarchy);
 
+            docId = parentId;
```

Comment:
In a try-get pattern, out params should be set to default when the return value is false. In this case `docId` and `documentPath` should be `0` and `null` respectively if the return value is `false`. `documentPath` looks like it is correct, but can you make sure to set `docId` to `0` if you are returning false? I would set it to `0` here, and then only set it to `parentId` on line 442.